alertmanager/README.md

```diff
@@ -0,0 +1,27 @@
+# Monitoring Kubecost with Prometheus/Thanos AlertManager
+
+![Example Alert](./images/example-alert.png)
+
+## Usage
+
+To start testing these alerting rules, you can apply these additional configs to your Kubecost primary cluster. Eventually they will need to be applied to each cluster Kubecost is monitoring.
+
+```bash
+helm upgrade -i kubecost cost-analyzer \
+  --repo https://kubecost.github.io/cost-analyzer/ 
+  -f values.yaml
+```
+
+To receive notifications when the alerting rules fire, you need to setup Prometheus AlertManager. You can do so by creating the following configmap, ensuring that `.Values.prometheus.alertmanager.enabled=true`, and then restarting the prometheus-server or thanos-query pod.
```

Comment:
Is there a reason we are creating the configmap outside of helm instead of including the alertmanager.yaml in the values file?

https://github.com/kubecost/cost-analyzer-helm-chart/blob/136b7c9cbf468beab513c5e38b090ad9b250de6f/cost-analyzer/charts/prometheus/templates/alertmanager-configmap.yaml#L12

Reply:
Awesome catch @srpomeroy!! I've made the update so it's all in the `values.yaml` now :)